app/app/VotingList.php

```diff
@@ -73,6 +73,9 @@ public function toSearchableArray()
             'summary' => $this->vote->voteCollection->summary?->text,
             'session_id' => $this->vote->session?->id,
             'session_display_title' => $this->vote->session?->display_title,
+            'members' => $this->members->mapWithKeys(function ($member) {
```

Comment:
We should add `members` to `makeAllSearchableUsing` in order to eagerly load the relation.

Reply:
I tried adding `members` to the list of relations to load eagerly, but for some reason, it didn’t import/index any voting lists at all. It did not error though. I’d probably need to take a look at the generated SQL, as I do not really have a clue why that’s happening…